### PR TITLE
Add EyeControl to Microsoft.Windows.Setting.Accessibility

### PR DESCRIPTION
**Pull request by Trenly**
- **Add EyeControl to Microsoft.Windows.Setting.Accessibility**
- **Add tests**

* Related to #53 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-dsc/pull/127)

### CODE REVIEW

File: resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1 [hunk 1]
```diff
@@ -891,6 +897,37 @@ class FilterKeys {
     }
 }
 
+[DscResource()]
+class EyeControl {
+    [DscProperty(Key)] [Ensure] $Ensure
```
Comment (Gijsreyn) [inline resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1:902]: Can we move forward with the `$Exist` property for DSC versions and higher?

Comment (Trenly) [reply resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1:902]: Boolean values can’t be used as a key

**Review by ryfu-msft — APPROVED** [review]
:shipit:

**Pull request merged**